ONSdigital/dp-code-list-api
# Feature/healthcheck

### What

- Converted to go module
- Replaced go-ns/log -> log.go
- Use new healthcheck library
- Use latest dp-graph and fixed the link creation / update according to the new DB models.
- Created models tests and improved / extended API tests
- Add Neo4J checker

### How to review

- Make sure code changes make sense
- Make sure unit tests pass
- You can try the health endpoint `GET http://localhost:22400/health`
- Check that the API endpoints work, and the returned links are as expected.

### Who can review

Anyone.
